developer_guides/topology2/topology2.rst

```diff
@@ -1332,6 +1332,51 @@ You can use the ``-P`` switch to convert a 2.0 configuration file to the 1.0 con
 
    alsatplg <-D args=values> -P input.conf -o output.conf
 
+Split topologies
+****************
+
+Linux kernel can load multiple topologies, a topology for a single function.
+This feature is useful when a single device is disabled in specific productuds. To achieve this, you need to split
+the topology into multiple tplg files. The split topology files should be named as follows:
```

Comment:
I think this is more useful to handle the mix and match nature of things?
Or actually to support SDCA setups with standardized components.

Reply:
Yeah, lets call out this is to support SDCA standards.

Reply:
Changed to "This feature is useful to support SDCA setups with standardized components. And no need to create topologies for every new product."